packages/base-types/src/node/stream.ts

```diff
@@ -15,10 +15,16 @@ export enum TraceStreamAction {
   PAUSE = 'PAUSE',
 }
 
-export interface StepPorts<BP extends Record<string, BasePort> = { [PortType.PREVIOUS]?: BasePort; [PortType.PAUSE]?: BasePort }>
-  extends BaseStepPorts<{ [PortType.NEXT]: BasePort } & BP, []> {}
+export interface StepBaseBuiltInPorts extends BuiltInNextPort {}
 
-export interface Step<Data = StepData, Ports = StepPorts> extends BaseStep<Data, BasePortList, Ports> {
+export interface StepDefaultBuiltPorts extends StepBaseBuiltInPorts {
```

Comment:
should we define these here and make them the default? Since we should only have these ports for the alexa version of this step

Reply:
I guess we need the on general as well, alexa just extends this type, makes it more strict about previous